api/src/main/java/uk/gov/hmcts/payment/api/configuration/security/SpringSecurityConfiguration.java

```diff
@@ -73,7 +73,10 @@ public void configure(WebSecurity web) {
                 "/webjars/springfox-swagger-ui/**",
                 "/swagger-resources/**",
                 "/v2/**",
-                "/refdata/**",
+                "/refdata/methods",
```

Comment:
Hi @vigneshpushkaranHMCTS - Why we need to add this separately? Is there any specific reason for this?

Reply:
Hi @aravindan89 , The new API method is added to existing controller which by default not have authentication, As discussed with @YasharRahvar @AyanDuttaHMCTS  , We added authentication to only new api to not affect existing API.